mappings/net/minecraft/predicate/entity/EntityEffectPredicate.mapping

```diff
@@ -15,6 +15,7 @@ CLASS bg net/minecraft/predicate/entity/EntityEffectPredicate
 	FIELD a EMPTY Lbg;
 	FIELD b effects Ljava/util/Map;
 	METHOD a create ()Lbg;
+	METHOD a effect (Lagn;)Lbg;
```

Comment:
Haven't checked the method, but should this be `getEffect`?

Reply:
Notice the return type is bg, which is the predicate itself. It is a setter named in builder style.

Reply:
If it's a static method, then perhaps create(arg) or createEffect?

Reply:
This is an instance method.
![image](https://user-images.githubusercontent.com/7806504/50898174-36c5e480-13c3-11e9-9395-9fcd00281c72.png)
The subclass `a` is per-effect qualification, but enigma fails to rename the subclass and I thus did not rename it.

Reply:
I'd say "addEffect" or "withEffect" here, if only because, even if builder style, this is *not* a builder - it's the class itself.

Reply:
`withPresence`? This method just makes the predicate require the presence of the status effect in the parameter.

Reply:
Eh, let's keep it as-is for now. It's not a particularly important class anyway.